JustSaying.TestingFramework/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AWSSDK" version="2.3.52.0" targetFramework="net452" />
+  <package id="NLog" version="4.2.0" targetFramework="net452" />

```

Comment:
NLog V 4.2.0 is that the other projects in the solution use. I take the view that an inconsistent package version is worse than being consistently a minor release or 2 behind.
We should update to latest at some point, in a different PR?